### ISSUE TITLE
create README

### ISSUE THREAD

**Issue by giovana-morais** [giovana-morais/color_cluster#6]
it must contain a detailed explanation about the following cases:

1. argument is a folder
- if this happens, images should be grouped by their similarity and saved in different folders to represent the clusters. or maybe just output a list with the clusters participants.

2. argument is a file
- if only a file is provided, code should return only the clusters of that image.

3. a file and a folder are passed
- basically, the program should get the image and return the images related ONLY to that image.
